portGps:
  x: 0
  y: -1.1557

starboardGps:
  x: 0
  y: 1.1557

imu:
  x: -0.2413
  y: 0
